frontend/view-edit-client/client-home/client-section.component.tsx

```diff
@@ -75,6 +75,23 @@ function getCss() {
       align-items: center;
     }
 
+    & .client-section-content > form > div > label > span {
```

Comment:
This CSS rule and all of the other ones you added would be significantly easier to understand and maintain if you put a css class onto the span and then referenced that css class directly.